doc/en/components/grids/_shared/column-resizing.md

```diff
@@ -0,0 +1,538 @@
+---
+title: {Platform} {ComponentTitle} Column Resizing - {ProductName}
+_description: Start using {Platform} {ComponentTitle} Column Resizing in order to change the grid column width in an instant. {Platform} drag resizing has never been so easy. Try for free!
+_keywords: {Platform}, {ComponentTitle}, {ComponentName}, {ProductName}, Infragistics
+mentionedTypes: [{ComponentApiMembers}]
+sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
+---
+
+# {Platform}  {ComponentTitle} Column Resizing
+
+With deferred grid column resizing, the user will see a temporary resize indicator while the {Platform} drag resizing operation is in effect. The new grid column width is applied once the drag operation has ended.
+
+## {Platform}  {ComponentTitle} Column Resizing Example
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:550px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-resizing-sample" alt="{Platform} {ComponentTitle} Column Resizing Example">
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:550px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-column-resizing" alt="{Platform} {ComponentTitle} Column Resizing Example">
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:510px" 
+           data-demos-base-url="{environment:lobDemosBaseUrl}" 
+           iframe-src="{environment:lobDemosBaseUrl}/{ComponentSample}-resizing" alt="{Platform} {ComponentTitle} Column Resizing Example">
+</code-view>
```

Comment:
remove 2 out of 3 code-view blocks and Component build flags because will use the same routing path for Grid, TreeGrid, HierarchicalGrid in samples.